### ISSUE TITLE
Mouse events don't trigger on parent or any children on touch devices / emulator

### ISSUE THREAD

**Issue by thelucre** [mauricius/vue-draggable-resizable#83]
Using version `v1.7.1`

Are click events being hijacked on all children? I've tried putting a click handler on every level of the following template code, but no click event will fire when clicking within the VDR parent area. 

I also set a `document` level event listener with the same issue above. Any clicks within the VDR parent element do not trigger a call back. 

I am using the library to grab and pan a map image, and I need map pins to be clickable. Any thoughts on how to achieve this? 

Thanks!

```pug
.Maps
  v-ons-toolbar
    .center Maps

  .map-container
    vue-draggable-resizable(
      :resizable="false"
      :parent="true"
      :minw="1500"
      :minh="1000"
      :h="1000"
      :x="250"
      :y="100"
      :drag-cancel="'.no-drag'"
    )
      visual(image="/static/img/fpo-map.png")

      .pins
        MapPin(@click="onClick" x="800" y="300") 
```

**Comment by mauricius** [comment]
I tried to replicate your issue using the following code, but everything is working fine.

```
<VueDraggableResizable :x="50" :y="50" :w="400" :h="400" :minw="0" :minh="0" :parent="true">
    <p>Component</p>
    <button @click="dosomething" @touchstart="dosomething">Click me</button>
</VueDraggableResizable>
```

The `click` event is handled correctly on Desktop and `touchstart` is handled correctly with the device emulator in Chrome. Do I miss something from your example?

**Comment by becrespi** [comment]
Yes this is strange, inside of a component `VueDraggableResizable`, all click event seems to be deactivated on mobile and same for scroll. Like mauris say to replace onclick event, `touchstart` work fine but for scroll i don't have found solution
Someone have an idea ?

**Comment by mauricius** [comment]
Touch events on the component use the `prevent` [event modifier](https://vuejs.org/v2/guide/events.html#Event-Modifiers) in order to prevent default browser behavior. This avoids undesired behavior, for example having the page scrolling while dragging an element. Having said that, can you please provide a concrete example where things don't work as you expected?

**Comment by becrespi** [comment]
Hi, thank for your response, for an example of my problem check [this link](https://codesandbox.io/s/387yw4nlm)
Scroll on the list works well on desktop but on touch devices, I can't scroll at all, do you have any solution ?

**Comment by becrespi** [comment]
Great thanks ! 
Do you have any idea of time before the next update ? Like next week, next month or more ?

**Comment by mauricius** [comment]
I'm going to fix it this weekend, so it's just a matter of few days 👍

**Comment by Lemenxzy** [comment]
Hi, is it finished?

**Comment by Lemenxzy** [comment]
Hi, is it fixed?

**Comment by TitanFighter** [comment]
Above your comment you can read:

> **mauricius** closed this in [cbbe7c3](https://github.com/mauricius/vue-draggable-resizable/commit/cbbe7c34f60a22498f3515b3917723b50a4060e0) 14 days ago

So yes, it is fixed.
